### ISSUE TITLE
Labels overlap

### ISSUE THREAD

**Issue by massimo-cassandro** [frappe/charts#171]
#### Steps to Reproduce:

```
{
	"height": 250,
	"title": "Fatturati totali per network",
	"type": "pie",
	"data": {
		"labels": ["Cisalpina Tours", "Bluvacanze", "Travelbuy", "Primarete Viaggi e Vacanze", "Blunet", "Uniposte"],
		"colors": ["#7ED321", "#007bff", "#c00", "#fc0", "#FDA5EE", "#9c6d1e"],
		"datasets": [{
			"name": "ago 2018",
			"values": [3848, 3415, 3500, 2704, 1295, 3339]
		}]
	}
}
```

![schermata 2018-05-07 alle 12 01 10](https://user-images.githubusercontent.com/1047169/39696626-8fa48b5c-51ee-11e8-8832-474030795525.png)

Frappé Charts version: 1.1.0

**Comment by ssuess** [comment]
is there a solution to this? I have the same problem...

**Comment by mojtabaahn** [comment]
same

**Comment by scmmishra** [comment]
Fixed this in the recent release... labels will automatically fit the width of the chart. ([ref](https://github.com/frappe/charts/commit/94c835115b946bad3dc383023c8a8bcb0abe73e2))

Also a recent PR will allow truncating of labels. PR: https://github.com/frappe/charts/pull/234

**Comment by mojtabaahn** [comment]
@scmmishra thanks
so it's gonna be available on v1.2.4 right?

**Comment by scmmishra** [comment]
The PR has been merged.

>  so it's gonna be available on v1.2.4 right?

Yes, along with a few other fixes. I am hoping to release it soon.
